app/src/main/res/menu/menu.xml

```diff
@@ -4,6 +4,6 @@
     <item
         android:id="@+id/bt_connect"
         android:title="@string/connect"
-        android:icon="@drawable/ic_wifi_tethering_black_24dp"
+        android:icon="@drawable/ic_portable_wifi_off_black_24dp"
```

Comment:
Yes, I also thought about that for a while... but then I would have had to change the test cases again. And actually I think it is more intuitive this way (like it works e.g. in iOS control center, and also Android?) otherwise when you are connected and the "wifi-off" icon is shown it feels somehow strange. 🤔